DQMOffline/Trigger/plugins/Tau3MuMonitor.h

```diff
@@ -0,0 +1,72 @@
+#ifndef TAU3MUMONITOR_H
```

Comment:
hi @rmanzoni - please include the package name in the include guard (DQMOFFLINE_TRIGGER_TAU3MUMONITOR_H

Reply:
ok, doing now